code/game/objects/structures/flora.dm

```diff
@@ -720,6 +720,83 @@
 	mood_change = -6
 	timeout = 5 MINUTES
 
+/obj/structure/flora/tree/srm
+	name = "Montagne's Oak"
+	icon = 'icons/obj/flora/chapeltree.dmi'
+	icon_state = "churchtree"
+	desc = "A sturdy oak tree imported directly from Illestren the homeworld of the Saint-Roumain Militia. The soil was carfuly transfered from the same place it was planted. A apple tree branch has been grafted onto it"
+	pixel_x = -16
+	max_integrity = 200
+	bound_height = 64
+	var/health = 0
+	var/lastcycle = 0
+	//Decides where the apple gets dropped
+	var/apple_direction = SOUTH
+	//Determines the health gained/lost when feeding the tree this chem
+	var/list/healthchems = list(
+		/datum/reagent/consumable/ethanol/ash_wine = 0.8,
+		/datum/reagent/water = 0.1,
+		/datum/reagent/plantnutriment = 0.2,
+		/datum/reagent/medicine/earthsblood = 1,
+		/datum/reagent/water/holywater = 0.8,
+		/datum/reagent/medicine/cryoxadone = 0.3,
+		/datum/reagent/ammonia = 0.4,
+		/datum/reagent/saltpetre = 0.5,
+		/datum/reagent/ash = 0.2,
+		/datum/reagent/diethylamine = 0.5,
+		/datum/reagent/consumable/nutriment = 0.1,
+		/datum/reagent/consumable/virus_food = 0.1,
+		/datum/reagent/blood = -0.1,
+		/datum/reagent/consumable/ethanol = -0.1,
+		/datum/reagent/toxin = -0.2,
+		/datum/reagent/fluorine = -0.3,
+		/datum/reagent/chlorine = -0.3,
+		/datum/reagent/toxin/acid = -0.3,
+		/datum/reagent/toxin/acid/fluacid = -0.4,
+		/datum/reagent/toxin/plantbgone = -0.5,
+		/datum/reagent/napalm = -0.6,
+		/datum/reagent/hellwater = -1,
+		/datum/reagent/liquidgibs = -0.2,
+		/datum/reagent/consumable/ethanol/demonsblood = -0.8,
+		/datum/reagent/medicine/soulus = -0.2
+	)
+
+/obj/structure/flora/tree/srm/Initialize()
+	START_PROCESSING(SSobj, src)
+	create_reagents(300, DRAINABLE | OPENCONTAINER)
```

Comment:
is this supposed to be drainable? if it IS, then OPENCONTAINER covers that by default. if not, then you should use OPENCONTAINER & ~(DRAINABLE), to get OPENCONTAINER "minus" DRAINABLE

Reply:
My original plan was for you to have to extract the Bactria from the tree so that's why it originally was drain able